loadbalancer/doc.go

```diff
@@ -1,24 +1,29 @@
 /*
-Package loadbalancer implements predicates and filter which will match for different backends
-in a round-robin fashion.
+Package loadbalancer implements load balancer algorithms that are applied by the proxy.
 
-First parameter to LBGroup, lbDecide and LBMember defines a group which determines the set of possible routes to match.
+roundRobin Algorithm
 
-lbDecide's second parameter is the number of members in a loadbalancer group.
+    The roundRobin algorithm does proxy requests round robin to
+    backend endpoints. It has a mutex to update the index and will
+    start at a random index
 
-LBMember's second parameter is 0-based index of the route among the other routes in the same group.
+random Algorithm
+
+    The random algorithm does proxy requests at random to backend
+    endpoints.
+
+consistentHash Algorithm
+
+    The consistentHash algorithm choose backend endpoints by hashing
+    client data with hash function fnv.New32. The client data is the
+    client IP, which might be looked up from X-Forwarded-For header.
```

Comment:
Is the phrasing with "might" on purpose? Currently, it relies on the X-Forwarded-For header, falling back to the remote address.

Reply:
maybe better then "might" is "will" ?

Reply:
How about "The client data is the client IP, inferred from the X-Forwarded-For header when available, otherwise from the remote IP of the request."?